datetime/src/main/java/com/fasterxml/jackson/datatype/jsr310/deser/InstantDeserializer.java

```diff
@@ -130,17 +134,31 @@ public class InstantDeserializer<T extends Temporal>
      * Flag set from
      * {@link com.fasterxml.jackson.datatype.jsr310.JavaTimeFeature#NORMALIZE_DESERIALIZED_ZONE_ID} to
      * determine whether {@link ZoneId} is to be normalized during deserialization.
+     *
+     * @since 2.16
      */
     protected final boolean _normalizeZoneId;
 
+    /**
+     * Flag set from
+     * {@link com.fasterxml.jackson.datatype.jsr310.JavaTimeFeature#ALWAYS_ALLOW_STRINGIFIED_TIMESTAMPS}
+     * to determine whether numeric strings are interpreted as numeric timestamps
+     * (enabled) nor not (disabled) in addition to an explicitly defined pattern.
```

Comment:
Should we replace "pattern" with "custom DateTimeFormatter" as you added to the changelog?